_release/bat/base_bat/base_bat_test.go

```diff
@@ -311,6 +317,155 @@ func TestDeleteAllInstances(t *testing.T) {
 	}
 }
 
+//Launch and instance create form an image with a customized cloud init
```

Comment:
The comments for all BAT test cases have 3 paragraphs.  The first just has a one line description of the test, the second has a description of what the test does and the 3rd is the expected result.  Look at some of the other BAT tests for an example.